environment/bashrc/.bashrc_cray

```diff
@@ -82,51 +78,14 @@ arch=`uname -m`
 
 # Use modules found in the draco directory
 sysname=`/usr/projects/hpcsoft/utilities/bin/sys_name`
-#if test -d $HOME/modulefiles/$sysname; then
-#  export ucmf=$HOME/modulefiles
-#  module use --append ~/modulefiles/$sysname
-#else
-  module load user_contrib
-#fi
-module load friendly-testing
-if [[ ${SLURM_JOB_PARTITION} == "knl" ]]; then
-  module swap craype-haswell craype-mic-knl
```

Comment:
so going forward do we load `draco/intel19` and then do the swap by hand?

Reply:
That is the idea.  We load the 'production' set of tools by default.  Anyone can swap out individual components as they see fit.  We can create other _super modules_ for alternate sets (e.g. intel-17 or gcc-9).